protos/sources/vega/markets.proto

```diff
@@ -283,6 +283,10 @@ message Market {
   string linear_slippage_factor = 13;
   // Quadratic slippage factor is used to cap the slippage component of maintenance margin - it is applied to the square of the slippage volume.
   string quadratic_slippage_factor = 14;
+  // The ID of the market this market succeeds
+  string parent_market_id = 15;
+  // The fraction of the insurance pool of the parent market this market inherits (range 0 through 1)
+  string insurance_pool_fraction = 16;
 }
```

Comment:
Shouldn't we annotate these as optional for clarity, as not every market will have these?